aiohttp/connector.py

```diff
@@ -948,27 +950,31 @@ async def _create_connection(
     @staticmethod
     @functools.lru_cache(None)
     def _make_ssl_context(verified: bool) -> SSLContext:
+        """Create SSL context.
+
+        This method is not async-friendly and should be called from a thread
+        because it will load certificates from disk and do other blocking I/O.
+        """
         if verified:
             return ssl.create_default_context()
-        else:
-            sslcontext = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
-            sslcontext.options |= ssl.OP_NO_SSLv2
-            sslcontext.options |= ssl.OP_NO_SSLv3
-            sslcontext.check_hostname = False
-            sslcontext.verify_mode = ssl.CERT_NONE
-            try:
-                sslcontext.options |= ssl.OP_NO_COMPRESSION
-            except AttributeError as attr_err:
-                warnings.warn(
-                    "{!s}: The Python interpreter is compiled "
-                    "against OpenSSL < 1.0.0. Ref: "
-                    "https://docs.python.org/3/library/ssl.html"
-                    "#ssl.OP_NO_COMPRESSION".format(attr_err),
-                )
-            sslcontext.set_default_verify_paths()
-            return sslcontext
+        sslcontext = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+        sslcontext.options |= ssl.OP_NO_SSLv2
+        sslcontext.options |= ssl.OP_NO_SSLv3
+        sslcontext.check_hostname = False
+        sslcontext.verify_mode = ssl.CERT_NONE
+        try:
+            sslcontext.options |= ssl.OP_NO_COMPRESSION
+        except AttributeError as attr_err:
+            warnings.warn(
+                "{!s}: The Python interpreter is compiled "
+                "against OpenSSL < 1.0.0. Ref: "
+                "https://docs.python.org/3/library/ssl.html"
+                "#ssl.OP_NO_COMPRESSION".format(attr_err),
+            )
```

Comment:
Changed in version 3.6: OpenSSL 0.9.8, 1.0.0 and 1.0.1 are deprecated and no longer supported. In the future the ssl module will require at least OpenSSL 1.0.2 or 1.1.0.

I think we can drop this